app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -7,6 +7,12 @@ angular.module('miq.util').factory('chartsMixin', function() {
       ({col1: theMoment.format('h:mm A'), col2: data[0].value + ' ' + data[0].name});
   };
 
+  var dailyTimeTooltip = function (data) {
+    var theMoment = moment(data[0].x);
+    return _.template('<div class="tooltip-inner"><%- col1 %> <%- col2 %></div>')
+    ({col1: theMoment.format('DD/MM/YY'), col2: data[0].value + ' ' + data[0].name});
```

Comment:
trying to understand what's going on in lines 12 and 13 - how do they represent a single statement? (especially line 13)